### ISSUE TITLE
High CPU Usage (Render Processes)

### ISSUE THREAD

**Issue by atom0s** [Microsoft/vscode#34499]
- VSCode Version: 1.16.1 (2017-09-14 - Shell: 1.7.3 - Renderer: 58.0.3029.110 - Node: 7.9.0 - Architecture ia32)
- OS Version: Windows Server 2012 R2 Datacenter w/GUI

Steps to Reproduce:

1. Install latest version of VS Code (32bit or 64bit)
2. Run it.

Reproduces without extensions: Yes

The rendering processes will each take up ~40% CPU just idling. During startup it will max the CPU the entire time starting, it takes upto 2 minutes for the UI to finally show. After that it keeps the CPU pinned at 60-100% usage doing literally nothing.

Server is a fresh install of 2k12 R2 and VSCode is a fresh install with no extensions. 
Happens on both 32bit and 64bit builds.

**Comment by atom0s** [comment]
Here is a pic of the initial start, ~30 seconds of this:
https://i.imgur.com/vE9z642.png

And afterward, a 2nd renderer launches:
https://i.imgur.com/OFynD8W.png

**Comment by apap04** [comment]
can reproduce. :feelsgood:

**Comment by atom0s** [comment]
I was able to resolve this partially with the setup I was using. The "machine" in question is a Hyper-V VM image of Windows Server 2012 R2 Datacenter. The image was set to only use 1 virtual CPU which limited it to ~2% of the host CPUs max resources. Bumping up the count to 8 instead of 1 fixed the lack of VSCode even loading past the empty screen I showed above. However, the CPU usage is still very high during startup and idle afterward.
